pkg/util/tracing/tracingpb/recording.go

```diff
@@ -219,6 +219,25 @@ func (r Recording) FindLogMessage(pattern string) (string, bool) {
 	return "", false
 }
 
+// FindLastLogMessage returns the last log message in the recording that matches
+// the given regexp. The bool return value is true if such a message is found.
+//
+// This method strips the redaction markers from all the log messages, which is
+// pretty inefficient.
+func (r Recording) FindLastLogMessage(pattern string) (string, bool) {
+	re := regexp.MustCompile(pattern)
+	for i := len(r) - 1; i >= 0; i-- {
+		sp := r[i]
+		for j := len(sp.Logs) - 1; j >= 0; j-- {
+			msg := sp.Logs[j].Msg().StripMarkers()
```

Comment:
Can you explain why `StripMarkers` is needed?

Reply:
There was a similar function just above looking for logs, so I followed the same pattern. No other particular reason.